src/server_manager/install_scripts/install_server.sh

```diff
@@ -402,6 +402,7 @@ function set_hostname() {
   # We have more than one to try in case one starts failing
   # (e.g. https://github.com/Jigsaw-Code/outline-server/issues/776).
   local -ar urls=(
+    'https://ifconfig.me/ip'
```

Comment:
Why is this needed?
Also, who runs ifconfig.me, and are they trusted?

Reply:
It looks like ifconfig.me runs on GCP, which means it won't work in China or Iran: https://host.io/ifconfig.me